lib/server.js

```diff
@@ -59,6 +60,9 @@ function serverFactory(core, messageHandler, Logger) {
             logger.info('proxyDHCP server is listening on %s:%s'.format(self.address, self.inPort));
             self.server.setBroadcast(true);
         });
+
+        var DHCPPoller = new IscDhcpLeasePoller({}, {});
+        DHCPPoller.run();
```

Comment:
I like the adding it as a config option to be able to turn off/on. If we do that then we should also add the config option to turn off/on the ARP Poller.